File: Strings/ExcelColTitle.py
class Solution(object):
    def convertToTitle(self, columnNumber):
        if columnNumber<=26: return chr(columnNumber-1 + ord("A"))

        res = ""
        while columnNumber>0:
            columnNumber -= 1
            currCol = chr((columnNumber%26) + ord("A"))
            res = currCol + res
            columnNumber = columnNumber // 26

        return res

File: Strings/ExcelColTitle.java
// Leetcode 168

public class ExcelColTitle{
    public String convertToTitle(int columnNumber) {
        String res = "";
        int n = 'A';

        while (columnNumber>0){
            columnNumber--;
            char currCol = (char)((columnNumber%26) + n);
            res = currCol + res;
            columnNumber = Math.floorDiv(columnNumber, 26);
        }
        return res;
    }
     public static void main(String[] args) {

    }
}